admin-dev/themes/default/template/controllers/modules_positions/list_modules.tpl

```diff
@@ -169,6 +169,18 @@
 		</div>
 	</div>
 	<div class="col-lg-3">
+		<div class="panel">
+			<h3><i class="icon-eye-open"></i> {l s='Live Edit'}</h3>
+			{if $live_edit}
+				<p>{l s='You have to select a shop to use Live Edit'}</p>
+			{else}
+				<p>{l s='Click here to be redirected to the front office of your shop where you can move and delete modules directly.'}</p>
+				<a class="btn btn-default _blank" href="{$url_live_edit|escape:'html':'UTF-8'}">
+					<i class="icon-edit"></i>
+					{l s='Run Live Edit'}
```

Comment:
Translations requires a domain to be properly organized on Crowdin.

ping @LouiseBonnard for follow up

Reply:
Indeed! @MahdiKhemakhem, this blog post on [Build](http://build.prestashop.com/news/new-translation-system-prestashop-17) explains everything, you should check on it to create proper strings of translation.